Documentation/connectors/saml.md

```diff
@@ -0,0 +1,152 @@
+# Authentication through SAML 2.0
```

Comment:
The documentation has been moved out of this repo.

Reply:
Done

Reply:
Will add the docs in the correct repo once this has been merged